### DISCUSSION TITLE
Leaving a lot of silence in my export

### DISCUSSION THREAD

**Discussion by WarpstarxG** [WyattBlue/auto-editor#480] [1. Help] [answered]
Hello
When I --export resolve even with no parameters it leaves ALOT of silence in the timeline. Some of the gaps are 1second or more.
I have tried audio thresholds and margin parameters and no matter what it just seems to cut very little silence out of my audio.
I have even exported with no parameters allowing for it to cut maximum silence and still so much dead space.

I love this plugin so much i'm just wondering if i'm doing something wrong?

I installed the newest update and still no such luck.

Any help would be greatly appreciated. 

Using davinci resolve
currently tried
auto-editor "filename.mp4" --export resolve
auto-editor "filename.mp4" --margin 0.2sec --export resolv
auto-editor "filename.mp4" --edit audio:-25dB --export resolve
it barely removes any silence at all

![example](https://github.com/WyattBlue/auto-editor/assets/174845034/a6d3a824-07af-432c-ac56-a7c612510a40)

**Comment by WyattBlue** [comment, marked as answer]
1. Set the threshold to a high dB value like `-9dB`. Low values like `-25dB` leave more cuts in.
1. Set the margin to 0 or even a negative value.
1. Make sure you're importing the Auto-Editor cuts and not the original video
